src/pfs/pfs.proto

```diff
@@ -239,6 +241,8 @@ message SubscribeCommitRequest {
   Commit from = 3;
   // Don't return commits until they're in (at least) the desired state.
   CommitState state = 4;
+  bool all = 5; // Return commits of all kinds (without this, aliases are excluded)
```

Comment:
I guess it would be overkill to make these a shared subproto

Reply:
Yeah, nested protos aren't very nice to use.